### PR TITLE
On Web, implement navigating back/forward with mouse buttons

### PR DESCRIPTION
**Pull request by emilk**
### What
By popular request

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using newly built examples: [rerun.io/viewer](https://rerun.io/viewer/pr/5792)
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/5792?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/5792?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/5792)
- [Docs preview](https://rerun.io/preview/fb21b7fbbebe58182dfe33276e81c59540393683/docs) 
- [Examples preview](https://rerun.io/preview/fb21b7fbbebe58182dfe33276e81c59540393683/examples) 
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

### CODE REVIEW

**Review by jprochazk — APPROVED** [review]
Nice!

**Comment by emilk** [comment]
I assume that means you have tested it? (I have no mouse to test it with)

**Comment by Wumpf** [comment]
just tried it with my back/forward button blessed mouse and it works (firefox, mac)

File: crates/re_viewer/src/app.rs [hunk 1]
```diff
@@ -1320,6 +1320,22 @@ impl eframe::App for App {
                 .add(egui_ctx.input(|i| i.time), seconds);
         }
 
+        #[cfg(target_arch = "wasm32")]
```
Comment (Wumpf) [inline crates/re_viewer/src/app.rs:1323]: actually why not also no native 🤔 
although tbf I would have expect those buttons to change selection in the selection history

**Pull request merged**